src/actions/actionQuickSort.js

```diff
@@ -0,0 +1,75 @@
+export function startQuickSort(...timeouts) {
+  for (var i = 0; i < timeouts.length; i++) {
+    clearTimeout(timeouts[i]);
+  }
+  return {
+    type: 'START_QUICK_SORT'
+  }
+}
+
+export function quickSort(sortArray, currentlyChecking, quickPivotIndex, quickLowIndex, quickHighIndex, quickPairsToSort, quickPriorPivots, iterations, stopQuickSort, isSorted = false, quickSwapping = false, quickSwappedIndices = []) {
+  var isRunning = true;
```

Comment:
I'm betting you inherited this code.  Whenever you have this many parameters to a function, it starts making more sense to pass an object.  That could allow you later on to use Object.assign to merge object properties together for the final return too for cleaner code.